src/extra/inputs.cpp

```diff
@@ -278,41 +279,29 @@ struct Mouse : public Base {
         reinterpret_cast<SDL_Window *>(_sdlWinVar->payload.objectValue);
 
     const auto hidden = _hidden.get().payload.boolValue;
-    if (hidden && !_isHidden) {
-      SDL_ShowCursor(SDL_DISABLE);
-      _isHidden = true;
-    } else if (!hidden && _isHidden) {
-      SDL_ShowCursor(SDL_ENABLE);
-      _isHidden = false;
+    if (hidden != _isHidden) {
+      SDL_ShowCursor(hidden ? SDL_DISABLE : SDL_ENABLE);
```

Comment:
lol my code was silly, I must have used copilot or something - I did it in a rush for sure.